### DISCUSSION TITLE
How to change all uppercased by MahApps fonts back to normal in the whole application?

### DISCUSSION THREAD

**Discussion by herczegzoltan** [MahApps/MahApps.Metro#4092] [Q&A] [answered]
I have  a MVVM application and after adding MahApps I can not change the uppercased fonts back to normal, like header of listview, buttons etc.

**Comment by punker76** [comment, marked as answer]
The header of a ListView can be changed like this

```xaml
<Style BasedOn="{StaticResource MahApps.Styles.GridViewColumnHeader}" TargetType="GridViewColumnHeader">
    <Setter Property="mah:ControlsHelper.ContentCharacterCasing" Value="Normal" />
</Style>
```

On a Button or any other ContentControl which uses this attached property this can be done like this

```xaml
<Button mah:ControlsHelper.ContentCharacterCasing="Normal" />
```

The `mah` namespace is `xmlns:mah="http://metro.mahapps.com/winfx/xaml/controls"`

**Reply by herczegzoltan** [reply]
Thank you!
